addons/l10n_be_pos_restaurant/models/pos_config.py

```diff
@@ -0,0 +1,64 @@
+from odoo import api, models
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    # def _create_tax_alcohol(self):
+    #     tax_ref = 'l10n_be_pos.tax_alcohol_luxury'
```

Comment:
Note that this is wrong because you can't share the same xmlid for records from different companies (and you need one account per company, at least by default)